### ISSUE TITLE
security: mariadb bump to 10.1.37

### ISSUE THREAD

**Issue by thelangley** [cloudfoundry/cf-mysql-release#218]
# Upgrade mariadb version to 10.1.37 to fix vulnerabilities
Vulnerabilities include:
- CVE-2018-3251
- CVE-2018-3156
- CVE-2018-3143
- CVE-2018-3174
- CVE-2016-9843
- CVE-2018-3282

See https://mariadb.com/kb/en/library/mariadb-10137-release-notes/ for more information

## Proposed solution/feature
Update references in the release for mariadb/mariadb-10.1.36.tar.gz to mariadb/mariadb-10.1.37.tar.gz

**Comment by menicosia** [comment]
Hi @thelangley,

Sorry for the delay on this. This will ultimately be covered in story [#161737247](https://www.pivotaltracker.com/story/show/161737247). Recently MariaDB made a change which broke our pipelines. We've filed a MariaDB bug: [MDEV-18059](https://jira.mariadb.org/browse/MDEV-18059). In the meanwhile, we've settled on a workaround, which will be covered in story [#163045333](https://www.pivotaltracker.com/story/show/163045333).

As soon as we've gotten to that story, we'll be unblocked from upgrading to MariaDB 10.1.37, and will be able to close this issue.

Thank you for pinging us!

--
Marco Nicosia
Product Manager
Pivotal Software, Inc
